app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -1213,7 +1281,7 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
                     isValid = true
                 }
             } catch (e: Exception) {
-                Log.w(LOG_TAG, "$FILETAG Error: ${e.message}", e)
+                Log.w(LOG_TAG_VPN, "$FILETAG Error: ${e.message}", e)
```

Comment:
Lines 1273 to 1284, I've seen it repeat in multiple places and files now. Why not abstract it out to a common method? 

Also, the exception... is it really required?

Also, simplify the nested `if..else` statements.

Reply:
Modified the if-else statement. 
abstracted out the common code. 
Modified the exception.

Reply:
Modified the if-else statement. 
abstracted out the common code. 
Modified the exception.